src/Common/QueryFuzzer.cpp

```diff
@@ -165,7 +164,6 @@ Field QueryFuzzer::fuzzField(Field field)
         {
             size_t pos = fuzz_rand() % arr.size();
             arr.erase(arr.begin() + pos);
-            std::cerr << "erased\n";
```

Comment:
Do you know if `stderr` is parsed elsewhere? It may be an incompatible change if so.